certified-connectors/Babybots/Readme.md

```diff
@@ -0,0 +1,12 @@
+## Babybots Connector
```

Comment:
Please refer below template link and update all required fields as mentioned in the link. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md